solutions/enterprise-java/adoption.adoc

```diff
@@ -50,8 +50,8 @@ Concentrate on coding web applications instead of solving scaling problems.
 
 ===== Get Started
 |=======
-|Watch a link:http://developers.redhat.com/video/vimeo/95291413[JBoss Data Grid Overview]
-|Learn more about link:http://developers.redhat.com/video/vimeo/28201194[Leveraging JBoss Data Grid as a local cache]
+|Watch a link:#{site.base_url}/video/vimeo/95291413[JBoss Data Grid Overview]
+|Learn more about link:#{site.base_url}/video/vimeo/28201194[Leveraging JBoss Data Grid as a local cache]
```

Comment:
The variable is not being expanded. Probably because the adoc file is missing ':awestruct-interpolate: true' from the top.